yt/frontends/arepo/tests/test_outputs.py

```diff
@@ -0,0 +1,55 @@
+"""
+Arepo frontend tests
+
+
+
+
+"""
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2015, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from collections import OrderedDict
+from yt.testing import requires_file
+from yt.utilities.answer_testing.framework import \
+    data_dir_load, \
+    requires_ds, \
+    sph_answer
+from yt.frontends.arepo.api import ArepoHDF5Dataset
+
+bullet_h5 = "ArepoBullet/snapshot_150.hdf5"
```

Comment:
I've staged this dataset on the jenkins test box so this test will run whenever we merge yt-4.0 with master. Note that this test *is not* running right now so if you can please make sure that the test runs correctly now.